### PR TITLE
Forms Automated Test - LPS-131000| master

### PR DESCRIPTION
**Pull request by wesleyroberts**

### CODE REVIEW

**Comment by CalumR23** [comment]
Hey @wesleyroberts you committed a commerce test on this PR as well. Please remove the commerce commits for further review.

**Comment by wesleyroberts** [comment]
Changes made

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path [hunk 1]
```diff
@@ -53,6 +53,11 @@
     <td>xpath=(//input[contains(@class,'form-builder-page-header-title')])[${key_titleNumber}]</td>
     <td></td>
 </tr>
+<tr>
+    <td>PAGE_DESCRIPTION_FIELD</td>
```
Comment (emanuelfisilva) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path:57]: Hi @wesleyroberts , please use the already existent path `Form#DESCRIPTION_FIELD`. Please, remove this.

**Comment by wesleyroberts** [comment]
Conflicts resolved.

**Comment by CalumR23** [comment]
@wesleyroberts please remove the commits not pertaining to the forms test

**Comment by wesleyroberts** [comment]
@CalumR23  changes made.

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:forward

**Pull request closed without merging**